classes/controllers/grid/users/reviewer/PKPReviewerGridHandler.inc.php

```diff
@@ -60,6 +63,41 @@ function __construct() {
 		);
 	}
 
+	/**
+	 * @copydoc PKPHandler::authorize()
+	 */
+	function authorize($request, &$args, $roleAssignments) {
```

Comment:
Incidentally, the `ReviewerGridHandler::authorize` functions in OJS and OMP are identical. If you don't mind tacking on a bit of slightly-out-of-scope clean-up, could you move them here to decrease the amount of app-specific code?

Reply:
:+1: